gems/sorbet-runtime/test/types/attached_class.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require_relative '../test_helper'
+
+module Opus::Types::Test
+  class AttachedClassTest < Critic::Unit::UnitTest
+    it 'can type self methods' do
```

Comment:
I might add another test that makes it clear that this does not raise when you don't return something that's not a valid value of type `AttachedClass`.

Reply:
Good idea, I've added one.